### ISSUE TITLE
How about removing your gem from rubygems?

### ISSUE THREAD

**Issue by KevinBongart** [will/bundle#3]
I agree with you that `gem install bundle` is annoying, but it would be less so if the user got an error about the gem not being found rather than the standard "installed" message.

**Comment by tom-lord** [comment]
I think a better solution would be to add a post-install message, explaining what had just happened.

For example, a warning about which version of bundle you installed would be nice. At the time of writing this:
gem install bundle  --> installs bundler-1.8.0.pre
gem install bundler --> installs bundler-1.7.13

**Comment by nkgm** [comment]
What's wrong with letting people realize it's bundle**r** and not taking away their chance to learn it properly? The existence of this gem is more confusing than anything else.

**Comment by tlunter** [comment]
Agreed with this.  Was installing `bundle` and it was installing 1.11 beta 2 which was causing all sorts of problems.

**Comment by tom-jin** [comment]
An error message would be more useful than this package.
If someone is mistakenly trying to update `bundler` by running `gem install bundle` the package is not updated when `bundle` is installed.

**Comment by meinac** [comment]
👍

**Comment by michaelkirk** [comment]
Better that this gem exists than something malicious with the same name.

https://news.ycombinator.com/item?id=11862217
